### ISSUE TITLE
Update text on Full Install page

### ISSUE THREAD

**Issue by oliver-phet** [phetsims/website#268]
We would like to update the text on the full install page once @jbphet has created separate OS versions of the installer with activities: http://phet.colorado.edu/en/get-phet/full-install

Text mockup here: https://docs.google.com/document/d/1314Ew34EmuVD6vYpNOoWs9Q1r8BtRGtv85NmckdHvMc/edit

**Comment by jbphet** [comment]
The installer-builder can now create the separate files.  Assigning to @aaronsamuel137 for the next steps.

**Comment by aaronsamuel137** [comment]
@jbphet, where are these files stored on figaro? I don't see them under phet.colorado.edu/installer. I just see the zip files. Shouldn't there be files for the installer with and without activities now?

**Comment by patricialoeblein** [comment]
I am glad this page is being revised because I know teachers get confused. Is there a separate issue for editing this page: http://phet.colorado.edu/en/get-phet  since this page is often selected from the homepage? The "Where can I save them.." row needs work based on comments from teachers and  For example, using "them" for "individual sims" is odd, but more importantly the text under the online column says " Flash applets cannot be saved. Java applications are automatically stored in your computer's WebStart cache, but cannot be moved." this is out of date.

**Comment by aaronsamuel137** [comment]
I've updated this page.

AND, I've added a new feature to the admin page "Edit Content Page". Just click on this link, and then you will see a list of all of our translatable pages. Now any admin can go in and edit the English text on pages if anything needs adjusting. See #118.

So, if anything else needs adjusting on the Installer page, try to do it here first before assigning to me. Some things you won't be able to do there, such as adding whole new paragraphs, or adjusting the layout.

Assigning to @oliver-phet for review.
